### PR TITLE
New package: autox-0.1.1

### PR DESCRIPTION
**Pull request by Vaelatern**

### CODE REVIEW

File: srcpkgs/autox/files/autox/run [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+USER=username
+
+exec 2>&1
+exec setsid -w agetty -a $USER -n -l /usr/bin/autox -o $USER tty7 38400 linux
```
Comment (xtraeme) [inline srcpkgs/autox/files/autox/run]: Use `service/conf` to override `USERNAME` and `TTY`:

```
[ -s conf ] && . ./conf
exec setsid -w agetty -a ${USER:=USERNAME} -n -l autox -o $USER $TTY ...
```

File: srcpkgs/autox/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'autox'
+pkgname=autox
+version=0.1.1
+revision=1
+wrksrc=${pkgname}-${version}-${pkgname}
+build_wrksrc=src
+build_style=gnu-makefile
+makedepends="pam-devel"
+depends="pam"
```
Comment (xtraeme) [inline srcpkgs/autox/template]: `/etc/pam.d/autox` must be part of `conf_files`.

File: srcpkgs/autox/files/autox/conf [hunk 3]
```diff
@@ -0,0 +1,2 @@
+USER=my_user
+TTY=tty7
```
Comment (xtraeme) [inline srcpkgs/autox/files/autox/conf]: `service/conf` files should not be provided by the xbps packages. Those files should be created and handled exclusively by the admin.

File: srcpkgs/autox/template (hunk 2 above)
Comment (Vaelatern) [inline srcpkgs/autox/template]: Done

File: srcpkgs/autox/files/autox/run [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+exec 2>&1
+[ -s conf ] && . ./conf
+: ${USER:=nobody}
+: ${TTY:=tty7}
+echo "USER: $USER"
```
Comment (xtraeme) [inline srcpkgs/autox/files/autox/run]: Please remove those two `echo` lines :-)

Comment (Vaelatern) [inline srcpkgs/autox/files/autox/run]: *doh*

**Pull request merged**